### DISCUSSION TITLE
Issue with github upstream url using another account

### DISCUSSION THREAD

**Discussion by archiesoft** [community/community#54122] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have been facing a weird issue. I created a new github account. I tried to change global config with new username, email etc and tried to create local config with remote url using new account.I removed old account settings from everywhere. However still, i am not able to push my code to new account upstream repo url.

I have checked thoroughly and still couldn't find out the problem, and can only push to upstream repo after changing for old account.
It always gives me error for new account repo:
git push -u origin 
remote: Repository not found.
fatal: repository 'https://github.com/<new_username>/<new_account_repo_url>.git/' not found
however works for  'https://github.com/<old_username>/<old_account_repo_url>.git/' (after changing with git remote add origin <old_account_repo_url>)

I even tried github desktop app and tried to signIn with new user, but it's browser oauth connection is not working and not able to login.

**Comment by orvn** [comment]
1. If you haven't already, add an [existing or new public key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/checking-for-existing-ssh-keys) to your Github account

2. In the new account, make sure your [email is verified](https://github.com/settings/emails)

3. Make sure there are no vestiges of your old username or account information in either of these places:

```bash
cat ~/.gitconfig
```

And in the case of specific projects
```bash
cd ~/my-project
cat .git/config
```
